test/unit/base.test.js

```diff
@@ -1275,6 +1275,23 @@ describe('CONVERT FUNCTION TESTS ', function() {
           done();
         });
     });
+
+    it('Should generate collection where folder name doesn\'t contain spaces when ' +
+      'not present in operation path', function (done) {
+      var openapi = fs.readFileSync(testSpec, 'utf8');
+      Converter.convert({ type: 'string', data: openapi }, { schemaFaker: true }, (err, conversionResult) => {
+        expect(err).to.be.null;
+        expect(conversionResult.result).to.equal(true);
+        expect(conversionResult.output.length).to.equal(1);
+        expect(conversionResult.output[0].type).to.equal('collection');
+        expect(conversionResult.output[0].data).to.have.property('info');
+        expect(conversionResult.output[0].data).to.have.property('item');
+        expect(conversionResult.output[0].data.item.length).to.equal(2);
+        expect(_.map(conversionResult.output[0].data.item, 'name')).to.include('pets');
+        expect(_.map(conversionResult.output[0].data.item, 'name')).to.include('pet/{petId}');
```

Comment:
Can we assert it together instead of extracting and mapping the array values twice?

Either we could create a variable and use it to assert inclusion of items. Like:
```
const itemsMap = _.map(conversionResult.output[0].data.item, 'name');
```

Or we could use the chai's `member` assertion like:
```
expect(_.map(conversionResult.output[0].data.item, 'name')).to.include.members(['pets', 'pet/{petId}']);
```

@VShingala Leaving it up to you to decide. (I'd recommend using `members` though)

Reply:
Addressed!

Reply:
Looks good!